deps/lua/src/lua_cjson.c

```diff
@@ -469,23 +469,19 @@ static void json_append_string(lua_State *l, strbuf_t *json, int lindex)
 
     str = lua_tolstring(l, lindex, &len);
 
-    /* Worst case is len * 6 (all unicode escapes).
-     * This buffer is reused constantly for small strings
-     * If there are any excess pages, they won't be hit anyway.
-     * This gains ~5% speedup. */
-    if (len > SIZE_MAX / 6 - 3)
+    if (len > SIZE_MAX - 3)
         abort(); /* Overflow check */
-    strbuf_ensure_empty_length(json, len * 6 + 2);
+    strbuf_ensure_empty_length(json, len + 2);
 
-    strbuf_append_char_unsafe(json, '\"');
+    strbuf_append_char(json, '\"');
```

Comment:
We can write the first char unsafe, because we have ensured len + 2, so it should be safe.

We can use this to trigger the extra-tests to run after adding the label.

```suggestion
    strbuf_append_char_unsafe(json, '\"');
```